package.json

```diff
@@ -14,8 +15,9 @@
     "inter": "^2.1.0",
     "markdown-it-mathjax3": "^4.3.2",
     "markdown-it-vuepress-code-snippet-enhanced": "github:IMB11/md-it-enhanced-snippets#dfb9fa2",
-    "vitepress": "^1.0.0-rc.33",
-    "vitepress-versioning-plugin": "^0.10.0",
-    "vue": "^3.4.0"
+    "patch-package": "^8.0.0",
+    "universalify": "^2.0.1",
```

Comment:
What is this used for?

Reply:
patch-package applies the patch, `universalify` is a dependency of the PR, and therefore is needed by the patch - obviously when the PR is merged, this can be removed.